crates/ty_project/src/metadata.rs

```diff
@@ -35,7 +34,6 @@ pub struct ProjectMetadata {
     /// knowing which files to watch for changes.
     ///
     /// The path ordering doesn't imply precedence.
-    #[cfg_attr(test, serde(skip_serializing_if = "Vec::is_empty"))]
```

Comment:
`bincode` doesn't support this attribute.

Reply:
@MichaReiser is this one public facing?

Reply:
No, not yet :) We only use serialization in tests (that's why `cfg_attr(test)`)